pkg/policy/repository_test.go

```diff
@@ -2044,19 +2029,16 @@ func (ds *PolicyTestSuite) TestMinikubeGettingStarted(c *C) {
 		RuleOrigin: map[CachedSelector]labels.LabelArrayList{cachedSelectorApp2: {nil}},
 	}
 
-	if equal, err := checker.DeepEqual(l4IngressPolicy, expected.Ingress.PortRules); !equal {
-		c.Logf("%s", logBuffer.String())
-		c.Errorf("Resolved policy did not match expected: \n%s", err)
-	}
+	require.Equal(t, expected.Ingress.PortRules, l4IngressPolicy, "Resolved policy did not match expected")
```

Comment:
Similar as above, use `assert.Equal` here and print the log buffer on failure

Reply:
done for this file and repository_deny_test.go as well.